mgit/index.js

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env node
+
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const path = require( 'path' );
+const meow = require( 'meow' );
+const mgit = require( './lib/index' );
+
+const cli = meow( `
+                     _ _   
```

Comment:
I assume you don't run the mgit in CLI :D 

![image](https://cloud.githubusercontent.com/assets/2270764/21816217/a33c916a-d75f-11e6-857d-e9c17f335b63.png)